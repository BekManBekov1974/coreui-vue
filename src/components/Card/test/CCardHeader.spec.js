import { mount } from '@vue/test-utils'
import Component from "../CCardHeader";
const ComponentName = 'CCardHeader'
const wrapper = mount(Component, {
  context: {
    props: {
      header: 'header',
      align: 'left',
      variant: 'primary',
      textVariant: 'white',
      borderVariant: 'solid'
    }
  },
  slots: {
    default: 'content goes here'
  }
})

describe(ComponentName, () => {
  it('has a name', () => {
    expect(Component.name).toMatch(ComponentName)
  })
  it('renders correctly', () => {
    expect(wrapper.element).toMatchSnapshot()
  })
});
