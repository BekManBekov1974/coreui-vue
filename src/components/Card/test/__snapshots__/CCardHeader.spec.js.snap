// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CCardHeader renders correctly 1`] = `
<div
  class="card-header text-left bg-primary border-solid text-white"
>
  <template>
    content goes here
  </template>
</div>
`;
